### ISSUE TITLE
EPIC 212772187 (Yoda)

### ISSUE THREAD

**Issue by George-Hookway** [George-Hookway/PBjammy#27]
![212772187 - 14 - MS - Echelle](https://github.com/user-attachments/assets/17dbb786-58cf-49a2-8482-f27994a2650e)
![212772187 - 14 - MS - Mode ID](https://github.com/user-attachments/assets/527dd42a-7bb7-4568-aaa6-c9ce41bc290f)
![212772187 - 14 - MS - Peakbagged Echelle](https://github.com/user-attachments/assets/40ea888f-fa00-4811-80cc-8ef3c45e3004)
![212772187 - 14 - MS - Peakbagging](https://github.com/user-attachments/assets/836a01f4-7f57-4da4-8d3c-d1cab65e34b6)

@nielsenmb @grd349

**Comment by nielsenmb** [comment]
Another one that's hard to tell. What epsilon does PBjam produce? 

I think this would benefit from an echelle with smaller symbols.

**Comment by grd349** [comment]
I would need convincing here!

**Comment by nielsenmb** [comment]
The end result is good, but the mode ID plots could be affected by the echelle sampling thing.

**Comment by George-Hookway** [comment]
Indeed, though it was then suffering from the common illness of wandering dnu :(

**Comment by grd349** [comment]
Can we have new plots? :) please

**Comment by George-Hookway** [comment]
![212772187 - 14 - MS - Echelle](https://github.com/user-attachments/assets/146f12b1-9424-43c7-9bc5-6e8631167dc4)
![212772187 - 14 - MS - Mode ID](https://github.com/user-attachments/assets/d1712b1b-4367-4fbb-8fc6-1817545d86f4)
![212772187 - 14 - MS - Peakbagged Echelle](https://github.com/user-attachments/assets/ce01f1f8-ce1b-4aa4-96f6-06483b390942)
![212772187 - 14 - MS - Peakbagging](https://github.com/user-attachments/assets/d4ec8c1d-1268-41ba-b492-f7e096c9587c)
![Reference](https://github.com/user-attachments/assets/90bbe47a-b8fe-4e4f-a695-c2ec58a03be6)

**Comment by George-Hookway** [comment]
From what I can see, H1_exp seems to be off. Could this be causing the potential issues with dnu?

**Comment by George-Hookway** [comment]
Actually, a manual inspection of the second panel in the mode ID spectrum shows that the spacing between peaks is slightly wider at higher frequencies. This would then mean the mode ID echelle is fine.

Again a case of two separate bumps in the spectrum. Binary??? Perhaps not :(    (Or perhaps so)
